### DISCUSSION TITLE
Musl support

### DISCUSSION THREAD

**Discussion by petm5** [ptitSeb/box86#712] [Q&A]
Is musl support planned for the future of Box86? Right now it does not compile on a musl host (as seen in ~~#327~~ #713). x86 binaries compiled for musl do not seem to work either when box86 is compiled on a glibc host.

**Comment by ptitSeb** [comment]
What would be the use case of musl build? to run x86 musl binary or run x86 non-musl one?

**Comment by petm5** [comment]
The use case would be to run x86 musl binaries (from an arm musl host). Although it would be nice to be able to run x86 non-musl binaries as well.
